plutus-contract/src/Wallet/Types.hs

```diff
@@ -213,15 +215,25 @@ instance Pretty AddressChangeResponse where
 --   outputs at a specific address in a slot range.
 data AddressChangeRequest =
     AddressChangeRequest
-        { acreqSlotRange :: SlotRange -- ^ The slot range
-        , acreqAddress   :: Address -- ^ The address
+        { acreqSlotRangeFrom :: Slot
+        , acreqSlotRangeTo   :: Slot
+        , acreqAddress       :: Address -- ^ The address
         }
         deriving stock (Eq, Generic, Show, Ord)
         deriving anyclass (ToJSON, FromJSON)
 
+-- | The earliest slot in which we can respond to an 'AddressChangeRequest'.
+targetSlot :: AddressChangeRequest -> Slot
+targetSlot = succ . acreqSlotRangeTo
```

Comment:
Shouldn't this be `acreqSlotFrom`?

Reply:
No, we have to wait until the entire interval has passed, so that we can return all the txns that happened in this interval.

Reply:
Ah. I see.